exaudfclient/base/script_options_parser/ctpg/script_option_lines_ctpg.cc

```diff
@@ -97,14 +134,16 @@ constexpr parser option_parser(
             >= [](auto o) { return std::string(o.get_value()); },
         option_value(not_semicolon)
             >= [](auto o) { return std::string(o.get_value()); },
-        option_value(whitespaces)
-            >= [](auto o) { return std::string(o.get_value()); },
-        option_value(semicolon_escape)
-            >= [](auto o) { return std::string(";"); },
+        option_value(whitespace_escape_seq)
+            >= [](auto o) { return std::string(convert_whitespace_escape_seq(o.get_value())); },
+        option_value(escape_seq)
+            >= [](auto es) { return convert_escape_seq(es.get_value()); },
         option_value(option_value, not_semicolon)
             >= [](auto&& ov, auto v) { return std::move(ov.append(v.get_value())); },
-        option_value(option_value, semicolon_escape)
-            >= [](auto&& ov, auto v) { return std::move(ov.append(";")); },
+        option_value(option_value, whitespace_escape_seq)
```

Comment:
in theory, only the first whitespace needs escaping what are we doing now.

Reply:
YEs, that's the implementation. But as the lexer will detect the escaped white spaces, I need to handle this case here, so that the escaped white spaces are added "as is" to the option value. Please have a look to the test:
https://github.com/exasol/script-languages/blob/bb976a41170c576bf6728c599b0a8eaeaf3ce6f3/exaudfclient/base/script_options_parser/ctpg/test/script_option_lines_test.cpp#L266

Reply:
ah, overlooked that, you are right